src/pages/Overview.js

```diff
@@ -114,10 +114,11 @@ function Overview() {
 
     if (((account || {}).cdps || {}).length) {
       buildCdpOverview();
-      showBanner({
-        banner: 'claim',
-        props: { colName: 'NAME', amount: '5.55', symbol: 'SYM' }
-      });
+      // TODO: when uncommented, this causes far to many re-renders, but does not cause infinite loop
+      // showBanner({
+      //   banner: 'claim',
+      //   props: { colName: 'NAME', amount: '5.55', symbol: 'SYM' }
+      // });
     }
```

Comment:
Calling `showBanner` here triggers a lot of renders, but it doesn't break the page like in `Presentation.js`